src/xx-apt

```diff
@@ -140,6 +140,12 @@ if [ "$TARGETARCH" = "riscv64" ] && [ "$(xx-info vendor)" = "debian" ]; then
   echo "deb [ arch=riscv64 ] http://ftp.ports.debian.org/debian-ports sid main" >>/etc/apt/sources.list.d/riscv64-sid.list
 fi
 
+if [ "$TARGETARCH" = "loongarch64" ] && [ "$(xx-info vendor)" = "debian" ]; then
+  apt-get update
+  apt-get install -y debian-ports-archive-keyring
+  echo "deb [ arch=loongarch64 ] http://pkg.loongnix.cn/loongnix http://pkg.loongnix.cn/loongnix" >>/etc/apt/sources.list.d/loongarch64.list
```

Comment:
Use only official sources by default. Debian seems to have them under ports, looks like alpine does not? Use of "pkg.loongnix.cn" can be enabled by user defining env variable. Eg. `XX_APT_REPO_URL_LOONGARCH64`.